pkg/subctl/cmd/gather.go

```diff
@@ -119,10 +147,10 @@ func gatherDiscovery(status *cli.Status, dataType string) error {
 	return nil
 }
 
-func gatherBroker(status *cli.Status, dataType string) error {
+func gatherBroker(status *cli.Status, clientSet kubernetes.Interface, dataType string, clustername string) error {
 	switch dataType {
 	case Logs:
-		status.QueueSuccessMessage("No logs to gather on Broker")
+		status.QueueWarningMessage("No logs to gather on Broker")
```

Comment:
```suggestion
```

Reply:
Will address this while adding log collection from Broker logic